components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/filter/ExpressionNode.java

```diff
@@ -55,4 +56,12 @@ public void setAttributeValue(String attributeValue) {
 
         this.attributeValue = attributeValue;
     }
+
+    public String getSubAttributeValue() {
```

Comment:
Follow the same pattern as name.GivenName

Reply:
Splitting the subAttributeValue from the attirbuteValue is now done in the service layer.